client/src/layout/config/chains.json

```diff
@@ -1,14 +1,16 @@
 [
   {
-    "title": "Gemini II",
+    "title": "Gemini III",
     "urls": {
-      "api": "https://squid.gemini-2a.subspace.network/graphql"
+      "api": "https://squid.gemini-3b.subspace.network/graphql",
+      "page": "gemini-3b"
```

Comment:
It's actually `gemini-3c`, I was re-using the same URL, but moving forward I will probably create a new droplet and a new URL for each iteration

Reply:
could you update this one to `gemini-3c`?